heron/common/src/cpp/network/client.cpp

```diff
@@ -181,8 +181,10 @@ void Client::OnNewPacket(IncomingPacket* _ipkt) {
 
   if (_ipkt->UnPackString(&typname) != 0) {
     Connection* conn = static_cast<Connection*>(conn_);
-    LOG(FATAL) << "UnPackString failed from connection " << conn << " from hostport "
+    LOG(ERROR) << "UnPackString failed from connection " << conn << " from hostport "
```

Comment:
Yeah, it makes sense to add a counter for dropped packets in another PR.
There is a real issue in our DC, the cause of the corruption is unknown yet.